Assignment-2/src/main/java/application/MainClass.java

```diff
@@ -0,0 +1,76 @@
+package application;
+
+import java.util.List;
+import java.util.Scanner;
+
+import models.Display;
+import models.UserClass;
+import services.ServicesHelper;
+import services.SortServiceHelper;
+
+/**
+ * Main class.
+ */
+public final class MainClass {
+  private MainClass() {
+
+  }
+
+  /**
+   * Main function.
+   * @param args represents args
+   */
+  public static void main(String[] args) {
+    final String filePath = "users.json";
+
+    // load users from the file
+    final List<UserClass> users = ServicesHelper.loadUsers(filePath);
```

Comment:
not every statement should require comment if the code is clean.

Reply:
will remove it